README.md

```diff
@@ -1,2 +1,133 @@
+   - [export_accounts](#export_accounts)
+   - [export_offers](#export_offers)
+   - [export_trustlines](#export_trustlines)
+- [History Archive Commands](#history-archive-commands)
+   - [export_ledgers](#export_ledgers)
+   - [export_transactions](#export_transactions)
+   - [export_operations](#export_operations)
+ - [Stellar Core Commands](#stellar-core-commands)
+   - [export_ledger_entry_changes](#export_ledger_entry_changes)
+
+Every command accepts a `-h` parameter, which provides a help screen containing information about the command, its usage, and its flags.
+
+### Bucket List Commands
+
+These commands use the bucket list in order to ingest large amounts of data from the history of the stellar ledger. If you are trying to read large amounts of information in order to catch up to the current state of the ledger, these commands provide a good way to catchup quickly. However, they don't allow for custom start-ledger values. For updating within a user-defined range, see the Stellar Core commands.
+
+#### export_accounts
+
+```bash
+> stellar-etl export_accounts --end-ledger 500000 --output exported_accounts.txt
+```
+
+This command exports accounts, starting from the genesis ledger and ending at the ledger determined by `end-ledger`. This command exports the point in time state of accounts, meaning that the exported data represents the account information as it was at `end-ledger`.
+
+#### export_offers
+
+```bash
+> stellar-etl export_offers --end-ledger 500000 --output exported_offers.txt
+```
+
+This command exports offers, starting from the genesis ledger and ending at the ledger determined by `end-ledger`. This command exports the point in time state of offer, meaning that the exported data represents the offerbook as it was at `end-ledger`.
+
+#### export_trustlines
+
+```bash
+> stellar-etl export_trustlines --end-ledger 500000 --output exported_trustlines.txt
+```
+
+This command exports trustlines, starting from the genesis ledger and ending at the ledger determined by `end-ledger`. This command exports the point in time state of trustlines, meaning that the exported data represents the trustline information as it was at `end-ledger`.
+
+### History Archive Commands
+
+These commands export information using the history archives. This allows users to provide a start and end ledger range. The commands in this category export a list of everything that occurred within the provided range.
+#### export_ledgers
+
+```bash
+> stellar-etl export_ledgers --start-ledger 1000 \
+--end-ledger 500000 --output exported_ledgers.txt
+```
+
+This command exports ledgers within the provided range. Both `start-ledger` and `end-ledger` are included in the export process.
+
+#### export_transactions
+
+```bash
+> stellar-etl export_transactions --start-ledger 1000 \
+--end-ledger 500000 --output exported_transactions.txt
+```
+
+This command exports transactions within the provided range. This range is inclusive.
```

Comment:
It's a bit awkward to specify that this range is inclusive, when all of the ranges are inclusive. I'd specify at the top that all ranges are inclusive, to be clear.